src/main/java/TaskManager.java

```diff
@@ -0,0 +1,50 @@
+public class TaskManager {
+    private static int taskNo = 0;
+    final private static int maxTasks = 100;
```

Comment:
Perhaps this should be [uppercase, as it is a constant](https://se-education.org/guides/conventions/java/basic.html#naming)?
```suggestion
    private static final int MAX_TASKS = 100;
```